tests/baselines/reference/scopeClassDeclaration.errors.txt

```diff
@@ -0,0 +1,7 @@
+tests/cases/compiler/scopeClassDeclarations.ts:4:11 - error TS1472: A class declaration cannot appear in statement-only position in 'ES2015' or higher.
+
+4     class C {} // runtime error if transpiled to ES2015 or above: Unexpected token class
+            ~
+
```

Comment:
Hm, shouldn't there be an error message on this line? Did the way we do baselines change?

Reply:
> Since this is my first contribution, I am not sure about the test coverage. In particular, I am not sure on how to generate the `.error.txt` file which I need to generate by running directly

Ah, I see, you grabbed this from TypeScript's direct output. You should just run

```sh
gulp baseline-accept
```

add that to git, and let us review that.

Reply:
Oh ok I see, thanks!

Reply:
I think I have got this right.

The first time I created the error file manually since it was not created by the command, as I needed to run tsc with a specific configuration (due to the target check in `checker`)

By removing the check the error file has been created.